Sources/TSFFutures/BatchingFutureOperationQueue.swift

```diff
@@ -1,99 +1,90 @@
+//===----------------------------------------------------------------------===//
+//
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
-// See http://swift.org/LICENSE.txt for license information
-// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
 
+import Dispatch
 import Foundation
 
-import TSCUtility
-
-
 /// Run the given computations on a given array in batches, exercising
 /// a specified amount of parallelism.
 ///
 /// - Discussion:
 ///     For some blocking operations (such as file system accesses) executing
 ///     them on the NIO loops is very expensive since it blocks the event
 ///     processing machinery. Here we use extra threads for such operations.
-public struct LLBBatchingFutureOperationQueue {
-
-    /// Threads capable of running futures.
-    public let group: LLBFuturesDispatchGroup
-
-    /// Queue of outstanding operations.
-    @usableFromInline
-    let operationQueue: OperationQueue
-
+public class LLBBatchingFutureOperationQueue {
```

Comment:
🤔 I think this would be cleaner if we added an internal protocol and used that for accessing whichever implementation is in use.

Reply:
After a discussion with @vlm we decided to get rid of operation queue based implementation entirely, so it is simplified now.